src/config.ts

```diff
@@ -100,6 +100,7 @@ export default {
       'husband',
       'obviously',
       'simple',
+      'of course'
```

Comment:
For `ts` we usually add a trailing command so that next time, it doesn't look like someone edited a line just to add a comma. But don't worry this is not a blocker to get your PR accepted and merged 🤓

Reply:
Oh alright! Sorry about that! I'll make sure to take care of it next time. Appreciate the feedback.

Reply:
No problem, I didn't document it anywhere, so my fault - also I am not sure if the any of our tools picks it up either, it is not going to break anything, so no worries 🤓

Reply:
I haven't worked on `ts` as much so I'll try to pick it up soon.

Reply:
No problem, great to have you contribute to our project, thank you 🤓